api/stacks/deploy.go

```diff
@@ -138,22 +131,3 @@ func getUserRegistries(datastore portainer.DataStore, user *portainer.User, endp
 
 	return filteredRegistries, nil
 }
-
-type cloneRepositoryParameters struct {
-	url   string
-	ref   string
-	toDir string
-	auth  *gitAuth
-}
-
-type gitAuth struct {
-	username string
-	password string
-}
-
-func cloneGitRepository(gitService portainer.GitService, cloneParams *cloneRepositoryParameters) error {
```

Comment:
👍 for removing this guy. Did you have a chance to run it against private and public repos to make sure it still works?